src/random_test.cpp

```diff
@@ -1607,20 +1608,63 @@ void Table::DeleteRandomRow(Thd1 *thd) {
       }
     }
   }
-
   std::string sql = "DELETE FROM " + name_ + " WHERE " +
-                    columns_->at(where)->name_ + "=" +
-                    columns_->at(where)->rand_value();
+                   columns_->at(where)->name_;
+  if (prob <= 4)
```

Comment:
use unequal probability select update and delete.

Reply:
Done